### PR TITLE
Added ProfileAbout component (redesign)

### PR DESCRIPTION
**Pull request by BronzW**
#2022 

~~Please don't merge this yet as I want to add responsiveness to this component.~~

### CODE REVIEW

**Comment by dk-raw** [comment]
Why did you made a pr then?

**Comment by goldyydev** [comment]
> ~~Please don't merge this yet as I want to add responsiveness to this component.~~

Does this mean that I can merge it or not?

**Comment by dk-raw** [comment]
> > ~Please don't merge this yet as I want to add responsiveness to this component.~
> 
> Does this mean that I can merge it or not?

I guess so

**Comment by BronzW** [comment]
Yes. I added responsiveness in https://github.com/benawad/dogehouse/pull/2075/commits/aa35422bba8cf033708f6fa0406b94bce6db58dd

File: kibbeh/src/ui/ProfileAbout.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            target="_blank"
+            rel="noreferrer"
+            
+          >{link.replace(/(^\w+:|^)\/\//, "")}</a>
+        </div>
+      }
+      {tags.map((props: UserBadgeLgProps) => <div className="mb-1"><UserBadgeLg {...props} /></div>)}
+    </div>
+  )
+}
```
Comment (goldyydev) [inline kibbeh/src/ui/ProfileAbout.tsx]:
```suggestion
}

```

**Review by goldyydev — CHANGES_REQUESTED** [review]

**Pull request merged**